var/spack/repos/builtin/packages/aspa/package.py

```diff
@@ -0,0 +1,72 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import glob
+
+
+class Aspa(MakefilePackage):
+    """A fundamental premise in ExMatEx is that scale-bridging performed in
+        heterogeneous MPMD materials science simulations will place important
+        demands upon the exascale ecosystem that need to be identified and
+        quantified.
+        tags = proxy-app
```

Comment:
sorry it has to be  `tags = ['proxy-app'] `, i'll correct that.

there are proxy apps which simulate experiments and integrating these app's  into spack.
and the tag was supposed to find the apps and to make it clear that it's a proxy app.